pkg/operator/vspherecontroller/checks/interface.go

```diff
@@ -2,18 +2,22 @@ package checks
 
 import (
 	"context"
+	"github.com/openshift/library-go/pkg/operator/configobserver/featuregates"
 	check "github.com/openshift/vmware-vsphere-csi-driver-operator/pkg/operator/vclib"
 )
 
 type CheckArgs struct {
-	vmConnection *check.VSphereConnection
-	apiClient    KubeAPIInterface
+	vmConnection        []*check.VSphereConnection
+	apiClient           KubeAPIInterface
+	featureGates        featuregates.FeatureGate
```

Comment:
Why are we supplying this featuregate when we aren't using it?

Reply:
We use it for a minimum of checking for multi vcenter: 
https://github.com/openshift/vmware-vsphere-csi-driver-operator/blob/5fc756c9695ccb96a5af5e869c42cad7bedef5c4/pkg/operator/vspherecontroller/vspherecontroller.go#L780
I wasn't sure if we want it to throw an error and prevent operator from functioning if the feature gate is not enabled.  I can also add other checks to verify fg set when we detect multi vcenter. If you have a preferred place, I can also put there.

Reply:
But that is not the place where `CheckArgs` is being used from. My point is - this field in `CheckArgs` is never used (at least as far as I can tell).

Reply:
oh, i thought that guy extended the CheckArgs.  Let me double check.  I'll clean up if its been removed (or use when i refactor next week based on our talks).

Reply:
I have cleaned this up.